setup.py

```diff
@@ -10,16 +10,16 @@
 
 install_requires = [
     'Django>=1.8,<1.12',
-    'djangorestframework==3.6.4', # Latest version that supports both Django 1.8 and 1.11
+    'djangorestframework==3.6.4',  # Latest version that supports both Django 1.8 and 1.11
     'elasticsearch>=2.4.1,<3',
-    'requests>=2.14,<2.15',
-    'urllib3>=1.21,<1.22',
+    'requests>2.19,<2.20',
+    'urllib3>1.21,<1.24',
```

Comment:
I don't actually see any direct use of `urllib3` in this repo. Perhaps we can remove this and just fall back on the version that gets installed by `elasticsearch`? Looking at the [requirements for that repo](https://github.com/elastic/elasticsearch-py/blob/2.5.0/setup.py#L15), it specifies `>=1.8,<2.0`, which is compatible both here and with the `1.22` version [specified in cfgov-refresh](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/libraries.txt#L52).